synthtool/gcp/templates/node_library/README.md

```diff
@@ -61,9 +61,8 @@ Google APIs Client Libraries, in [Client Libraries Explained][explained].
 ### Using the client library
 
 ```{{ metadata['repo']['language']|syntax_highlighter }}
-{{ metadata['quickstart'] }}{% if metadata['partials'] and metadata['partials']['quickstart_footer'] %}
-{{ metadata['partials']['quickstart_footer'] }}
```

Comment:
we've opted to instead start landing an executable sample format, making the `quickstart_footer` section of this template obsolete.